menas/Dockerfile

```diff
@@ -25,5 +25,6 @@ ADD ./src/main/resources/server.xml /tmp/server.xml
 EXPOSE 8080
 EXPOSE 8443
 EXPOSE 8009
-
+#Debugging
```

Comment:
This should probably be removed before merging.

Reply:
I think it depends. I thought it could help for troubleshooting locally

Reply:
Makes sense.

Reply:
Actually, the comment can be removed and left as is (`EXPOSE 5005`). When you then start the image you choose if to bind it or not